CMakeLists.txt

```diff
@@ -23,18 +23,31 @@ IF(RPM)
 	SET(CPACK_RPM_MXS_ADAPTER_PACKAGE_NAME "mariadb-columnstore-maxscale-cdc-adapters")
 	SET(CPACK_RPM_MXS_ADAPTER_FILE_NAME "${CPACK_RPM_MXS_ADAPTER_PACKAGE_NAME}-${CPACK_RPM_PACKAGE_VERSION}-${CPACK_RPM_PACKAGE_RELEASE}-${ENGINE_ARCH}-${RPM}.rpm")
 
-	# rename packets manually if CMAKE < 3.6.3
-	if(${CMAKE_VERSION} VERSION_LESS "3.6.3") 
```

Comment:
Given the large amount of code required to handle older CMake versions, wouldn't it be more convenient to raise the minimum version to 3.7.0? For example, MaxScale doesn't use the system CMake as the versions are wildly different. For this reason it is very convenient (and understandable) to require a certain minimum CMake version (3.7 seems to be pretty OK).